xcdat/spatial.py

```diff
@@ -441,12 +408,34 @@ def _get_longitude_weights(
         -------
         xr.DataArray
             The longitude axis weights.
+
+        Raises
+        ------
+        ValueError
+            If the there are multiple instances in which the
+            domain_bounds[:, 0] > domain_bounds[:, 1]
         """
         p_meridian_index: Optional[np.ndarray] = None
         d_bounds = domain_bounds.copy()
 
+        # check if there is more than one potential prime meridian cell
+        # there should only be one for rectilinear data
+        pmcells = np.where(domain_bounds[:, 1] - domain_bounds[:, 0] < 0)[0]
+        if len(pmcells) > 1:
+            raise ValueError(
+                "More than one longitude bound is out of order. Only one bound \n\
+                              spanning the prime meridian is permitted"
+            )
```

Comment:
```suggestion
        pm_cells = np.where(domain_bounds[:, 1] - domain_bounds[:, 0] < 0)[0]
        if len(pm_cells) > 1:
            raise ValueError(
                "More than one longitude bound is out of order. Only one bound "
                "value spanning the prime meridian is permitted in data on "
                "rectilinear grids."
            )
```

Removed comment and updated error message formatting and specifies rectilinear grids.